resources/views/layouts/datagrid/rows/entitylink.blade.php

```diff
@@ -6,6 +6,14 @@
     <?php return ?>
 @endif
 
+@if ($model instanceof \App\Models\EntityMention)
+    @if ($model->isEntity() && $model->entity->is_private || $model->isQuestElement() && $model->questElement->quest->entity->is_private || $model->isTimelineElement() && $model->timelineElement->timeline->entity->is_private || $model->isPost() && $model->post->entity->is_private)
```

Comment:
This crashes for non-admins with a mention to a private timeline or quest but where the element is visible to all. It needs all the checks on each element that "timeline" and "entity" is visible, or "quest" and "entity" is visible. It makes more sense to remove this from entitylink.blade.php and do a custom renderer in the datagrid, to avoid making the code more complicated.